v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/template/fx.tmpl

```diff
@@ -76,7 +77,8 @@
   func New{{.FxServer}}() interface{} {
     return func(p {{.FxServer}}Params) {{.FxServer}}Result {
       return {{.FxServer}}Result{
-        Procedures: Build{{.Procedures}}(p.Server),
+        {{if (unaryMethods .)}} UnaryProcedures: BuildUnary{{.Procedures}}(p.Server), {{end}}
+        {{if (streamMethods .)}}StreamProcedures: BuildStream{{.Procedures}}(p.Server), {{end}}
```

Comment:
I don't understand fx fully so I wasn't completely sure what to do here.

This section will look like:
```
UnaryProcedures: BuildUnary...(...),
```
if there are no stream procedures. Do we want this or:
```
UnaryProcedures: BuildUnary...(...),
StreamProcedures: nil,
```

Reply:
Meh, equivalent.